Resources/skeleton/crud/actions/edit.php.twig

```diff
@@ -17,7 +17,7 @@
     {
 {% block method_body %}
         $deleteForm = $this->createDeleteForm(${{ entity_singularized }});
-        $editForm = $this->createForm(new {{ entity_class }}Type(), ${{ entity_singularized }});
+        $editForm = $this->createForm({{ entity_class }}Type::class, ${{ entity_singularized }});
```

Comment:
maybe a check if php > 5.3 is installed or something like this or an option in the commands?